common/src/main/java/org/dash/wallet/common/ui/components/MyTheme.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2024 Dash Core Group.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.dash.wallet.common.ui.components
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.sp
+import org.dash.wallet.common.R
+
+object MyTheme {
+    val ToastBackground = Color(0xff191c1f).copy(alpha = 0.9f)
+
+    val Body2Regular = TextStyle(
+        fontSize = 14.sp,
+        lineHeight = 20.sp,
+        fontFamily = FontFamily.Default, // FontFamily(Font(R.font.inter)) // crashes,
+        fontWeight = FontWeight(400),
+        color = Color.White
+    )
```

Comment:
This is a start for a "stylesheet" file, there probably is a better way to do this, perhaps a standard way that it is done with Compose.

Reply:
Usually in JC you would set a MaterialTheme with a `typography` field, however, the names of standard text styles aren't going to match with our typography sheet in Figma, so it's better to have named styles as you did here.